Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -107,7 +111,7 @@ class Navigator {
                                                 config: NavigatorConfig(),
                                                 customConfig: customConfig)
         
-        historyRecorder = HistoryRecorderHandle.build(forHistoryFile: Navigator.historyFilePath?.absoluteString ?? "", config: configFactory)
+        historyRecorder = HistoryRecorderHandle.build(forHistoryFile: Navigator.historyDirectoryURL?.absoluteString ?? "", config: configFactory)
```

Comment:
@MaximAlien It should be `Navigator.historyDirectoryURL?.path`, not `Navigator.historyDirectoryURL?.absoluteString`. `url.absoluteString` returns a string in format `file:///...`